Writerside/topics/Quickstart-Up-And-Running.md

```diff
@@ -0,0 +1,463 @@
+## Overview
+
+By the end of this quickstart, you will be able to run your game, or a sample game in a multiplayer/online environment using AWS. Players will be able to:
+1. Log into your game
+2. Form a Party
+3. Request a Match
+4. Connect to a Server
+5. Play a game
+
+## Prerequisites
+
+What you will need before following the steps in this guide.
+
+* Access to Catena Tools
+* A Game with a client server model or the [Catena Lyra Demo](https://github.com/catenaTools/catena-lyra-demo)
+* The Catena Tools [AWS Infrastructure Template](https://github.com/CatenaTools/infrastructure/tree/main/aws)
+* [Terraform](https://www.terraform.io)
+* A Domain Name that you Control
+
+## Setting up Catena
+
+Prior to running the Terraform code, set up a hosted zone for the domain that you will use. In the example, this will be `catenatools.com.`
+
+See [Creating a public hosted zone](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingHostedZone.html). The Terraform configuration will create a few subdomain routes on this url.
+
+Next, the `infrastructure` repo and open it in your favorite editor. We will be working out of the `aws/` directory.
+
+```bash
+git clone git@github.com:CatenaTools/infrastructure.git
+cd infrastructure/aws
+```
+
+This template sets up a working [dokku](https://dokku.com) instance on the domain you select, along with a few additional settings to simplify deploying and running the platform, including an IAM role for `aws ssm` logins and an Elastic IP.
+
+### Configure Terraform
+
+The Terraform configuration defaults to using an s3 bucket to store it's state, this must be updated to an s3 bucket on your account. You may create this s3 bucket in the AWS Console.
+
+Open `main.tf` and modify the following portion to match your s3 bucket, or remove it to store state locally.
+
+```
+  backend "s3" {
+      bucket  = "catena-terraform-state"
+      key     = "updated-infra/terraform.tfstate"
+      region  = "us-east-1"
+      profile = "catena"
+  }
+```
+
+Be sure to replace `bucket` with your bucket's name, `region` with your region, and `profile` with your local [aws profile](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/configure/index.html).
+
+Next, we need to set some specific-to-you configurations.
+
+Create a `vars.tfvars` file similar to the following:
+
+```
+aws_profile="catena"                        # Your AWS Profile
+aws_region="us-east-1"                      # The region you are working in
+ec2_ami="ami-0c7217cdde317cfec"             # AWS Ubuntu 22.04 LTS AMI for your region
+ec2_instance_size="t2.small"                # Instance Size for the machine running the catena backend
+ssh_private_key_path="~/.ssh/id_rsa"        # Your private key
+ssh_public_key_path="~/.ssh/id_rsa.pub"     # Your public key
+route_53_hosted_zone_name="catenatools.com" # The name of the hosted zone in route53
+```
+
+### Provision the Infrastructure
+
+```bash
+terraform workspace switch -or-create dev
```

Comment:
```
terraform init
terraform workspace new dev
terraform workspace select dev
```

Reply:
if I understand correctly, switch -or-create does the same thing in one line

Reply:
Ah, interesting. TIL. I tried looking that up but didn't see any docs for it (I didn't look very hard)